src/Compilers/CSharp/CscCore/project.json

```diff
@@ -1,42 +1,45 @@
 {
   "dependencies": {
-    "System.AppContext": "4.0.0-beta-22816",
```

Comment:
I copied the csccore project.json to the vbccore project.json. I repushed (after rebasing and squashing).
I'll merge once testing passes.